app/tests/spec/views/complete_reset_password.js

```diff
@@ -273,7 +271,7 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
       });
 
       it('non-direct-access signs the user in and redirects to `/reset_password_complete` if broker does not say halt', function () {
-        var account;
+        var account; //eslint-disable-line no-unused-vars
```

Comment:
This one is kind of legit– we need `account` so that we can make the assertion [here](https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-7839367e19fc797c69e79d8581c483a3R310). But we can refactor this to make it better!

We can remove `account` and it's usage [here](https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-7839367e19fc797c69e79d8581c483a3R284) and just return `newAccount` directly. Then, for the test [here](https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-7839367e19fc797c69e79d8581c483a3R310) we can use some of [Sinon](http://sinonjs.org/docs/)'s reflection capabilities and do: 

```
 assert.isTrue(broker.afterCompleteResetPassword.calledWith(user.setSignedInAccount.returnValues[0]));
```